search-api/src/main/scala/no/ndla/searchapi/integration/ZipUtil.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Part of NDLA search-api
+ * Copyright (C) 2024 NDLA
```

Comment:
Fila er sikkert oppretta i 24 men siden pr er fra 25 tenker eg det er greit å ha det slik.
```suggestion
 * Copyright (C) 2025 NDLA
```